tests/fast-integration/monitoring-test/readme.md

```diff
@@ -0,0 +1,11 @@
+# Monitoring test
+
+This test covers the monitoring components.
```

Comment:
I'd personally like to see more details here, but I'll assume the actual testers have more context.
If they don't, we should add a little bit more info here :D

Reply:
The readme file was added to be aligned with the rest of the tests. However, it's not really valuable. You can reason about the tests just by looking at the code.